src/Formatters/FullyQualifiedFacades.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Tighten\TLint\Formatters;
+
+use PhpParser\Node;
+use PhpParser\Lexer;
+use PhpParser\Parser;
+use PhpParser\Node\Name;
+use PhpParser\NodeTraverser;
+use PhpParser\Node\Stmt\Use_;
+use PhpParser\Node\Stmt\UseUse;
+use Tighten\TLint\BaseFormatter;
+use PhpParser\Node\Stmt\Namespace_;
+use PhpParser\PrettyPrinter\Standard;
+use PhpParser\NodeVisitor\CloningVisitor;
+use Tighten\TLint\Concerns\IdentifiesFacades;
+
+class FullyQualifiedFacades extends BaseFormatter
+{
+    use IdentifiesFacades;
+
+    public const description = 'Import facades using their full namespace.';
+
+    public function format(Parser $parser, Lexer $lexer)
+    {
+        $traverser = new NodeTraverser;
+        $traverser->addVisitor(new CloningVisitor);
+
+        $oldStmts = $parser->parse($this->code);
+        $oldTokens = $lexer->getTokens();
+
+        $newStmts = $traverser->traverse($oldStmts);
+
+        if ($newStmts[0] instanceof Namespace_) {
+            $newStmts[0]->stmts = $this->transformFacadesToFullyQualified($newStmts[0]->stmts);
+        } elseif (count($newStmts) && ($newStmts[0] instanceof Use_ || $newStmts[0] instanceof UseUse)) {
+            $newStmts = $this->transformFacadesToFullyQualified($newStmts);
+        }
```

Comment:
Is there another situation other than these two? Where `count($newStmts)` returns empty or its first entry isn't an instance of `use`? If so, do we need to catch it or else we will be not transforming? If not, should this just be an `else`?